_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -59,6 +59,85 @@ The "Turn off" and "Remap" requirements from [Success Criterion 2.1.4][sc2.1.4]
 
 ## Test Cases
 
+<details>
+<summary>
+These Javascript and Style files are used in several examples:
```

Comment:
Any particular reason for using "Style files" instead of just "CSS files". I don't think I care much, just struck me as slightly odd.